src/app/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,21 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
+import { ApiService } from '../Services/api.service';
 
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.css'],
 })
-export class DashboardComponent {}
+export class DashboardComponent {
+  constructor(private apiService: ApiService) {}
+  ClockIn(): void {
+    this.apiService.PostClockIn().subscribe((response) => {
+      console.log(response);
+    });
+  }
+  ClockOut(): void {
+    this.apiService.PostClockOut().subscribe((response) => {
+      console.log(response);
```

Comment:
Starajmy się nie wrzucać console.logów do głównego brancha, jeśli nie ma takiej potrzeby (jak np. podczas logowania errorów)

Ciekawostka. Funkcje strzałkowe umożliwiają nam stworzenie one linera, kiedy wewnątrz funkcji wykonujemy tylko jedną czynność, wtedy kod wygląda w taki sposób:

  ClockIn() {
    this.apiService.PostClockIn().subscribe(**response => console.log(response)**);
  }